system-status.sh

```diff
@@ -38,3 +38,13 @@ log minikube_ssh journalctl --boot
 # filesystem status for host and VM
 log df -hT
 log minikube_ssh df -hT
+
+# fetch all logs from /var/lib/rook in the VM and write them to stdout
+log minikube_ssh sudo tar c /var/lib/rook | tar xvO
+
+# gets status of the Rook deployment
```

Comment:
can you add one more last thing to run `ceph -s` in toolbox pod?

```bash
kubectl exec --namespace=rook-ceph $(kubectl get pod -l "app=rook-ceph-tools" --namespace=rook-ceph -o jsonpath='{.items[0].metadata.name}') -- ceph -s
  cluster:
    id:     bbd16246-07fe-4f55-9de8-78e21c568c2e
    health: HEALTH_WARN
            client is using insecure global_id reclaim
            mon is allowing insecure global_id reclaim
 
  services:
    mon: 1 daemons, quorum a (age 3h)
    mgr: a(active, since 3h)
    osd: 1 osds: 1 up (since 3h), 1 in (since 3h)
 
  data:
    pools:   2 pools, 33 pgs
    objects: 14 objects, 51 B
    usage:   1.0 GiB used, 49 GiB / 50 GiB avail
    pgs:     33 active+clean
 
  io:
    client:   426 B/s rd, 0 op/s rd, 0 op/s wr
 
```

Reply:
isn't that part of the CephCluster output?

Reply:
Afaik Ceph CRD output won't provide more details but it will show whether cluster is healthy or not

Reply:
```
[ygupta@localhost ceph-csi]$ ../tbox.sh ceph status
  cluster:
    id:     4c398bda-eccf-4e65-bc78-e4cd633d33d4
    health: HEALTH_WARN
            mon is allowing insecure global_id reclaim
 
  services:
    mon: 1 daemons, quorum a (age 119m)
    mgr: a(active, since 119m)
    mds: myfs:1 {0=myfs-a=up:active} 1 up:standby-replay
    osd: 1 osds: 1 up (since 84m), 1 in (since 25h)
 
  data:
    pools:   4 pools, 97 pgs
    objects: 40 objects, 478 KiB
    usage:   1.0 GiB used, 19 GiB / 20 GiB avail
    pgs:     97 active+clean
 
  io:
    client:   852 B/s rd, 1 op/s rd, 0 op/s wr
```
```
[ygupta@localhost rbd]$ kubectl -n rook-ceph describe CephCluster
Name:         my-cluster
Namespace:    rook-ceph
Labels:       <none>
Annotations:  <none>
API Version:  ceph.rook.io/v1
Kind:         CephCluster
Metadata:
  Creation Timestamp:  2021-05-11T11:07:34Z
  Finalizers:
    cephcluster.ceph.rook.io
  Generation:  2
  Managed Fields:
    API Version:  ceph.rook.io/v1
    Fields Type:  FieldsV1
    fieldsV1:
      f:spec:
        .:
        f:cephVersion:
          .:
          f:allowUnsupported:
          f:image:
        f:crashCollector:
          .:
          f:disable:
        f:dashboard:
          .:
          f:enabled:
        f:dataDirHostPath:
        f:healthCheck:
          .:
          f:daemonHealth:
            .:
            f:mon:
              .:
              f:interval:
              f:timeout:
        f:mon:
          .:
          f:allowMultiplePerNode:
          f:count:
        f:storage:
          .:
          f:useAllDevices:
          f:useAllNodes:
    Manager:      kubectl-create
    Operation:    Update
    Time:         2021-05-11T11:07:34Z
    API Version:  ceph.rook.io/v1
    Fields Type:  FieldsV1
    fieldsV1:
      f:metadata:
        f:finalizers:
          .:
          v:"cephcluster.ceph.rook.io":
      f:spec:
        f:cleanupPolicy:
          .:
          f:sanitizeDisks:
        f:disruptionManagement:
        f:external:
          .:
          f:enable:
        f:healthCheck:
          f:daemonHealth:
            f:osd:
            f:status:
        f:mgr:
        f:monitoring:
        f:network:
          .:
          f:hostNetwork:
          f:provider:
          f:selectors:
        f:removeOSDsIfOutAndSafeToRemove:
        f:storage:
          f:config:
          f:storageClassDeviceSets:
      f:status:
        .:
        f:ceph:
          .:
          f:details:
            .:
            f:AUTH_INSECURE_GLOBAL_ID_RECLAIM_ALLOWED:
              .:
              f:message:
              f:severity:
          f:health:
          f:lastChecked:
        f:conditions:
        f:message:
        f:phase:
        f:state:
        f:storage:
          .:
          f:deviceClasses:
        f:version:
          .:
          f:image:
          f:version:
    Manager:         rook
    Operation:       Update
    Time:            2021-05-12T13:01:30Z
  Resource Version:  117909
  UID:               149f268f-d679-4fc9-b8d5-fe6950360623
Spec:
  Ceph Version:
    Allow Unsupported:  true
    Image:              ceph/ceph:v15
  Cleanup Policy:
    Sanitize Disks:
  Crash Collector:
    Disable:  true
  Dashboard:
    Enabled:           true
  Data Dir Host Path:  /var/lib/rook
  Disruption Management:
  External:
    Enable:  false
  Health Check:
    Daemon Health:
      Mon:
        Interval:  45s
        Timeout:   600s
      Osd:
      Status:
  Mgr:
  Mon:
    Allow Multiple Per Node:  true
    Count:                    1
  Monitoring:
  Network:
    Host Network:                          false
    Provider:                              
    Selectors:                             <nil>
  Remove OS Ds If Out And Safe To Remove:  false
  Storage:
    Config:                     <nil>
    Storage Class Device Sets:  <nil>
    Use All Devices:            true
    Use All Nodes:              true
Status:
  Ceph:
    Details:
      AUTH_INSECURE_GLOBAL_ID_RECLAIM_ALLOWED:
        Message:   mon is allowing insecure global_id reclaim
        Severity:  HEALTH_WARN
    Health:        HEALTH_WARN
    Last Checked:  2021-05-12T13:03:31Z
  Conditions:
    Last Heartbeat Time:   2021-05-11T12:10:05Z
    Last Transition Time:  2021-05-11T11:10:59Z
    Message:               Cluster progression is completed
    Reason:                ProgressingCompleted
    Status:                False
    Type:                  Progressing
    Last Heartbeat Time:   2021-05-11T11:11:37Z
    Last Transition Time:  2021-05-11T11:11:37Z
    Message:               Cluster created successfully
    Reason:                ClusterCreated
    Status:                True
    Type:                  Ready
    Last Heartbeat Time:   2021-05-11T11:11:39Z
    Last Transition Time:  2021-05-11T11:11:39Z
    Message:               Cluster progression is completed
    Reason:                ProgressingCompleted
    Status:                False
  Message:                 Cluster created successfully
  Phase:                   Ready
  State:                   Created
  Storage:
    Device Classes:
      Name:  hdd
  Version:
    Image:    ceph/ceph:v15
    Version:  15.2.11-0
```

Reply:
`ceph -s` is included now too